deploy/monitoring_reader_role.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: muo-monitoring-reader
+  namespace: openshift-monitoring
+rules:
+- apiGroups:
+  - ""
+  resources:
```

Also add `configmaps` to this section.

Since PR #273 was merged, it requires this otherwise it throws the following error:
```
configmaps \"serving-certs-ca-bundle\" is forbidden: User \"system:serviceaccount:openshift-managed-upgrade-operator:managed-upgrade-operator\" cannot get resource \"configmaps\" in API group \"\" in the namespace \"openshift-monitoring\"
```
```suggestion
  resources:
  - configmaps  
```